src/components/AutoComplete/AutoComplete.stories.tsx

```diff
@@ -0,0 +1,117 @@
+import { AutoComplete, AutoCompleteProps } from "./AutoComplete";
+import { Preview } from "@storybook/react";
+import { selectOptions } from "../Select/selectOptions";
+import { useEffect, useState } from "react";
+interface Props extends Omit<AutoCompleteProps, "options" | "children"> {
+  childrenType: "children" | "options";
+}
+const SelectExample = ({ childrenType, value, ...props }: Props) => {
+  const [selectedValue, setSelectedValue] = useState(value);
+  useEffect(() => {
+    setSelectedValue(value);
+  }, [value]);
+
+  if (childrenType === "options") {
+    return (
+      <AutoComplete
+        value={selectedValue}
+        onSelect={value => setSelectedValue(value)}
+        options={selectOptions}
+        {...props}
+      />
+    );
+  }
+  return (
+    <AutoComplete {...props}>
+      <AutoComplete.Group heading="Group label">
+        <AutoComplete.Item
+          value="content0"
+          icon="user"
+        >
+          Content0
+        </AutoComplete.Item>
+      </AutoComplete.Group>
+      <div>
+        <AutoComplete.Item value="content1">Content1 long text content</AutoComplete.Item>
+      </div>
+      <AutoComplete.Item
+        value="content2"
+        disabled
+      >
+        Content2
+      </AutoComplete.Item>
+      <AutoComplete.Item value="content3">Content3</AutoComplete.Item>
+      <AutoComplete.Item
+        value="content4"
+        label="Content4"
```

Comment:
when passing the value as label it doesn't show up in the search

Reply:
Fixed